clients/src/main/java/org/apache/kafka/clients/admin/QuorumInfo.java

```diff
@@ -64,6 +70,10 @@ public List<ReplicaState> observers() {
         return observers;
     }
 
+    public List<Node> nodes() {
```

Comment:
It seems better to have a `Map<Integer, Node>` since we know the IDs are unique.

Reply:
Fixed.